source/slang/slang-diagnostic-defs.h

```diff
@@ -108,6 +108,8 @@ DIAGNOSTIC(    70, Error, cannotMatchOutputFileToEntryPoint, "the output path '$
 
 DIAGNOSTIC(    80, Error, duplicateOutputPathsForEntryPointAndTarget, "multiple output paths have been specified entry point '$0' on target '$1'")
 
+DIAGNOSTIC(    81, Error, parametersAfterLoadReproIgnored, "Parameters after -load-repro [file] are ignored")
```

Comment:
Is there a reason for this choice? Why would parameters before loading the repro be allowed but not after?